media/js/src/views/quiz-validator.js

```diff
@@ -8,6 +8,38 @@ define([
      */
     var QuizValidator = Backbone.View.extend({
         /**
+         * This function returns false if all the checkboxes on the
+         * form that has class `cssClass` (default: 'quizblock-required')
+         * are unchecked. Otherwise, it returns true.
+         */
+        validateCheckboxForm: function($form, cssClass) {
+            if (typeof cssClass === 'undefined') {
+                cssClass = 'quizblock-required';
```

Comment:
If the default is "required", will the user need to add a dummy css class to turn validation off?